net/wireshark4/Portfile

```diff
@@ -30,12 +30,12 @@ if { ${os.platform} eq "darwin" && ${os.major} < 16 } {
     }
 }
 
-version         4.0.7
+version         4.0.8
 revision        0
 
-checksums       sha256  a79f7b04cbff823e30452abf4bcb86773d8583eb62d5f71f16c09f019f8a8777 \
-                sha1    ad2aa262e3376d72dfe1311530dee53a8c462c1b \
-                size    43113752
+checksums       rmd160  a1f4f6d4b6bd32dcfcdc4775d853394a17ab0f98 \
+                sha256  16663585c0ffefd5593a6628d4a20cc8241b9703b11283cfe71ead2b750888c8 \
```

Comment:
I don't mind adding rmd160, but please keep the sha1 hash - upstream provides sha256 and sha1 hashes in their announcements.

Reply:
Got it, sha1 hash is now back